pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -777,8 +738,8 @@ class CSWAP(Operation):
     Args:
         wires (Sequence[int]): the wires the operation acts on
     """
-    is_self_inverse = True
```

Comment:
Whoops what happened here? Seems to be unrelated to this PR? [This line should be deleted on the op-refactor branch anyways, but cleaner to leave it for now?]

Reply:
Oh! Good catch! I started this PR, noticed and deleted this line, then had a meeting, wanted to put it back in and notify you but forgot, submitted, and did not check everything in the end. I think this should be removed (and you seem to agree), but I agree this is not conceptually part of this PR. I will revert the change. Can you take care of implementing this in the op-refactor branch?